report-service/index.ts

```diff
@@ -1,7 +1,16 @@
 import Koa from 'koa';
 import bodyParser from 'koa-bodyparser';
+import mongoose from 'mongoose';
+import router from './routes';
 
-import router from './routes'
+const HOST = 'localhost';
+const PORT = '16717';
+const DATABASE = 'report-service';
+
+mongoose
+    .connect(`mongodb://${HOST}:${PORT}/${DATABASE}`, { useNewUrlParser: true })

```

Comment:
instead of 3 variables - we could use 1 (connection string) 
and we also could make it part of environment

Reply:
Good idea. I think we can use 'config' module here.

Reply:
I believe we'll use docker here and probably we will get port, database, etc. from env, that's why 3 variables here

Reply:
Then you should use them like environment variables.
process.env.HOST
process.env.PORT
process.env.DATABASE

But nevertheless you can use config module, where we can define some configurations and override them with environment variables later.
Something like they do here:
https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables

Reply:
moved variables to `.env` file, added new module `dotenv`